### PR TITLE
Update Favorite.js

### PR DESCRIPTION
**Pull request by yenandrew**
Working on the front end for the favorites page.

### CODE REVIEW

**Comment by yenandrew** [comment]
Wrong from HTML editor so will do multiple small commits

**Comment by yenandrew** [comment]
Merged more things from a separate branch

File: src/pages/Favorite.js [hunk 1]
```diff
@@ -1,7 +1,101 @@
...
-    render() {
-        return (<h1>FAVORITE PAGE !!</h1>)
-    }
-}
\ No newline at end of file
+function Favorites(props) {
+    const [recipeData, setRecipeData] = useState([])
+    const [restaurantData, setRestaurantData] = useState([])
+    
+    user = User.query.filter_by(username=current_user.username).first()
```
Comment (2v0v0v0) [inline src/pages/Favorite.js:9]: This is a back end function.

File: src/pages/Favorite.js [hunk 2]
```diff
@@ -1,7 +1,101 @@
...
+    current_user_restaurants = user.restaurants
+    current_user_restaurant_ids = []
+    
+    recipe_id = flask.request.json.get("recipe_ids")
+    restaurant_id = flask.request.json.get("restaurant_ids")
```
Comment (2v0v0v0) [inline src/pages/Favorite.js:14]: flask will not work in React frontend

**Pull request closed without merging**